spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin/ZipkinAutoConfiguration.java

```diff
@@ -53,24 +53,26 @@
  * @see DefaultZipkinRestTemplateCustomizer
  */
 @Configuration
-@EnableConfigurationProperties({ZipkinProperties.class, SamplerProperties.class})
+@EnableConfigurationProperties({ ZipkinProperties.class, SamplerProperties.class })
 @ConditionalOnProperty(value = "spring.zipkin.enabled", matchIfMissing = true)
 @AutoConfigureBefore(TraceAutoConfiguration.class)
 public class ZipkinAutoConfiguration {
 
 	@Bean
 	@ConditionalOnMissingBean
-	public ZipkinSpanReporter reporter(SpanMetricReporter spanMetricReporter, ZipkinProperties zipkin,
```

Comment:
revert every change to this method.. it is distracting as it is unrelated to this request